src/routes/_dashboard/assignUserToManager.tsx

```diff
@@ -0,0 +1,6 @@
+import assignUserToManager from '@/pages/Users/components/assignUserToManager';
+import { createFileRoute } from '@tanstack/react-router';
+
+export const Route = createFileRoute('/_dashboard/assignUserToManager')({
```

Comment:
Nie wiem czy potrzebna jest osobna strona do tego. Może dać to do edycji użytkownika?